### ISSUE TITLE
Support more svg syntaxes

### ISSUE THREAD

**Issue by WhenMelancholy** [ManimCommunity/manim#912]
## Description of bug / unexpected behavior

1. Manim will treat `c p1 p2 p3 p4 p5 p6...` as a Cubic Bezier curve command and there will be an error, but it should be treated as multiple curves.
2. Some svg files contain positions like `99.98.97` which means `99.98 0.97`, and it will cause `float()` throws a `ValueError`

## How to reproduce the issue

```svg
<svg t="1610093083946" class="icon" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="9062"
     width="200" height="200">
    <path d="M99.89.22M512 1024 c 282.784 0 512 -229.216 512 -512 0 -35.36 28.64 -64 64 -64 z"
          p-id="9063"></path>
</svg>
```
This svg file will cause the first and the second problem.

**Comment by dlnnlsn** [comment]
Actually, I think that the 'c' command does work correctly, but 's' does not. (Depending on what one considers to be "correct", but the code currently anticipates that 'c' could be followed by the points to specify multiple curves, but it doesn't do the same thing for 's'.)

**Comment by leotrs** [comment]
Hello @WhenMelancholy, thanks for reporting this!

We are well aware that the SVG parser in manim is severely lacking in features. Unfortunately, this is something that we have not had the bandwidth to address yet. If you are able, we would love a contribution or PR in this regard.

**Comment by MrMallIronmaker** [comment]
I'm also having similar issues - the ValueError one in particular. Is there a intuition from the dev team whether manim should parse the svg files on their own or whether it should interface with an svg library?

**Comment by leotrs** [comment]
@MrMallIronmaker the problem is that whether or not we use an external library, at the end we will need to manually build a Mobject from the SVG. Otherwise you won't be able to use Animations on your imported SVGs.

**Comment by MrMallIronmaker** [comment]
Right, right. If we limit ourselves to `<path>` type objects, it looks simple enough to effectively roll our own. I wonder though - there are a lot of other features in the SVG standard that we may have to support too. If there's an interface someone has already developed to go from SVG to points, then the translation from points to Mobject would be much simpler. I haven't seen that library, but I've only done a smidge of research so I can't conclude the library doesn't exist.

**Comment by leotrs** [comment]
It might well be the case that our job would be made easier by some library that already parses SVG files and converts them to some abstract representation. But in that case we would have to convert that abstract representation to our own...

**Comment by MrMallIronmaker** [comment]
A quick (ten minute) search led me to a lot of libraries that convert to a file format (PNG, PDF) or object another library expects (ReportLab, Cairo). It doesn't seem like there's a swiss-army-knife for SVG that these files build on.

Before I go digging deeper into the mess of libraries, I want to take stock of the kinds of SVG files people tend to need to import in Manim, so I'm in the middle of collecting all the "Manim breaks my SVG" posts from Discord. Then, I'll attach all those files to the Github issue about SVG tests (https://github.com/ManimCommunity/manim/issues/355). Whether I tackle SVG or someone else, there will be a set of images to test upon.

**Comment by leotrs** [comment]
We gathered some here https://github.com/ManimCommunity/manim/projects/8

**Comment by MrMallIronmaker** [comment]
This can be closed due to PR #915 - the file in question is included as a test named WatchTheDecimals.svg
